src-electron/generator/matter/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -262,6 +264,13 @@ async function asObjectiveCClass(type, cluster, options) {
         hash: { preserveAcronyms: preserveAcronyms },
       });
     }
+    // Use a custom prefix if specified.
+    if (options.hash.structTypePrefix) {
+      return (
+        options.hash.structTypePrefix +
+        `${cluster}Cluster${appHelper.asUpperCamelCase(type)}`
+      );
+    }
     return `MTR${cluster}Cluster${appHelper.asUpperCamelCase(type)}`;
```

Comment:
How about this instead:
```suggestion
    // Use a custom prefix if specified, otherwise default to "MTR" for backwards compat.
    return `${options.hash.structTypePrefix || "MTR"}${cluster}Cluster${appHelper.asUpperCamelCase(type)}`;
```